### PR TITLE
refcator,chore(egen) : refcatored code according to template guidelines

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes
1.changed variable REGION to LOCATION
2.added code to remove locally generated files in cleanup
3.removed UUID code and os module
4.performed linter test
5.followed template guidelines

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/migration/sdk-hyperparameter-tuning.ipynb [hunk 1]
```diff
@@ -82,12 +83,12 @@
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `Vertex AI Training`\n",
-        "- `Vertex AI Hyperparameter Tuning`\n",
+        "- Vertex AI Training\n",
+        "- Vertex AI Hyperparameter Tuning\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-hyperparameter-tuning.ipynb:87]: change: "Vertex AI Hyperparameter Tuning"
to: "Vertex AI hyperparameter tuning"

File: notebooks/official/migration/sdk-hyperparameter-tuning.ipynb [hunk 2]
```diff
@@ -543,7 +457,9 @@
         "\n",
         "The files `setup.cfg` and `setup.py` are the instructions for installing the package into the operating environment of the Docker image.\n",
         "\n",
-        "The file `trainer/task.py` is the Python script for executing the custom training job. *Note*, when we referred to it in the worker pool specification, we replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
+        "The file `trainer/task.py` is the Python script for executing the custom training job. \n",
+        "\n",
+        "*Note* : when referred to it in the worker pool specification, replaced the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-hyperparameter-tuning.ipynb:462]: Change Note:
- To: "Note*:  When `trainer/task.py` is referred to in the worker pool specification, the directory slash is replaced with a dot and the file suffix (`.py`) is dropped (`trainer.task`)."

File: notebooks/official/migration/sdk-hyperparameter-tuning.ipynb [hunk 3]
```diff
@@ -587,10 +503,10 @@
       "source": [
         "#### Task.py contents\n",
         "\n",
-        "In the next cell, you write the contents of the hyperparameter tuning script task.py. I won't go into detail, it's just there for you to browse. In summary:\n",
+        "In the next cell, you write the contents of the hyperparameter tuning script task.py. you won't go into detail, it's just there for you to browse. In summary:\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-hyperparameter-tuning.ipynb:506]: - Change: "you won't go into detail, it's just there for you to browse."
- To: "It's there for you to browse."

File: notebooks/official/migration/sdk-hyperparameter-tuning.ipynb [hunk 4]
```diff
@@ -587,10 +503,10 @@
...
         "\n",
         "- Parse the command line arguments for the hyperparameter settings for the current trial.\n",
-        " - Get the directory where to save the model artifacts from the command line (`--model_dir`), and if not specified, then from the environment variable `AIP_MODEL_DIR`.\n",
+        " - Get the directory where to save the model artifacts from the command line (`--model_dir`), and unspecified, then from the environment variable `AIP_MODEL_DIR`.\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-hyperparameter-tuning.ipynb:509]: rewrite: "Get the directory where the model artifacts from the command line (`--model_dir`) are to be saved. If unspecified, then from the environment variable `AIP_MODEL_DIR` is used.\n",   [Not sure if this is correct, but the current version isn't clear]

File: notebooks/official/migration/sdk-hyperparameter-tuning.ipynb [hunk 5]
```diff
@@ -831,9 +747,9 @@
         "\n",
         "-`package_uris`: This is a list of the locations (URIs) of your python training packages to install on the provisioned instance. The locations need to be in a Cloud Storage bucket. These can be either individual python files or a zip (archive) of an entire package. In the later case, the job service  unzip (unarchive) the contents into the docker image.\n",
         "\n",
-        "-`python_module`: The Python module (script) to invoke for running the custom training job. In this example, you  be invoking `trainer.task.py` -- note that it was not neccessary to append the `.py` suffix.\n",
+        "-`python_module`: The Python module (script) to invoke for running the custom training job. In this example, you  be invoking `trainer.task.py` -- note that it wasn't neccessary to append the `.py` suffix.\n",
         "\n",
-        "-`args`: The command line arguments to pass to the corresponding Pythom module. In this example, you  be setting:\n",
+        "-`args`: The command line arguments to pass to the corresponding Python module. In this example, you  be setting:\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-hyperparameter-tuning.ipynb:752]: - change: "you be setting" 
- to: "you are setting

File: notebooks/official/migration/sdk-hyperparameter-tuning.ipynb [hunk 6]
```diff
@@ -911,7 +827,7 @@
       "source": [
         "## Create a custom job\n",
         "\n",
-        "Use the class `CustomJob` to create a custom job, such as for hyperparameter tuning, with the following parameters:\n",
+        "Use the class `CustomJob` to create a custom job, such as  for hyperparameter tuning, with the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-hyperparameter-tuning.ipynb:830]: remove extra space after "as": "as for hyperparameter"

**Review by kittyabs — COMMENTED** [review]
suggested edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**